vispy/visuals/image.py

```diff
@@ -239,13 +244,13 @@ def _build_vertex_data(self):
         mgrid[..., 1] *= h
 
         quads[..., :2] += mgrid
-        tex_coords = quads.reshape(grid[1]*grid[0]*6, 3)
+        tex_coords = quads.reshape(grid[1] * grid[0] * 6, 3)
```

Comment:
FYI PEP8 E226 (whitespace around operator) is considered optional. I don't think `numpy` and `scipy` even enforce it, and we don't. You can keep the change here, though.